### DISCUSSION TITLE
[Feature Request] Ability to make headlines in the outliner

### DISCUSSION THREAD

**Discussion by mustafaalbazy** [vslinko/obsidian-outliner#218] [Archive]
It would be nice if we can add or make headlines H2, H3, H4 within the outliner, without the need to separate the outline then write the headline then start a new outline. As this takes out (or limits) the flexibility of moving section up/down.

Headlines are needed to make new ideas, topics...etc stand out, especially in a long page.

**Comment by vslinko** [comment]
With the new live preview mode you can do that:

```
- # heading
  - list item
```

**Reply by mustafaalbazy** [reply]
Thanks a lot man
